gunicorn/config.py

```diff
@@ -572,11 +570,11 @@ class WorkerClass(Setting):
         * ``gevent``   - Requires gevent >= 0.13
         * ``tornado``  - Requires tornado >= 0.2
 
-        Optionally, you can provide your own worker by giving gunicorn a
-        python path to a subclass of gunicorn.workers.base.Worker. This
-        alternative syntax will load the gevent class:
-        ``gunicorn.workers.ggevent.GeventWorker``. Alternatively the syntax
-        can also load the gevent class with ``egg:gunicorn#gevent``
+        Optionally, you can provide your own worker by giving Gunicorn a
+        Python path to a subclass of :class:`gunicorn.workers.base.Worker`.
```

Comment:
Okay, I'll revert this. It's [been used](https://github.com/benoitc/gunicorn/commit/33d35a1a665b4388a4d5cb77ea9903d0a2973d49#diff-9f531e15751f24da705e11f66baca7e3R10) by @matrixise though.